parent/pom.xml

```diff
@@ -241,11 +241,12 @@
 					<!-- set useUIThread=false for swtbot tests -->
 
 					<includes>
-						<include>**/AllTests.class</include>
-						<include>**/*AllTests*.class</include>
-						<include>**/*AllBotTests*.class</include>
-						<include>**/*TestSuite*.class</include>
+						<include>**/*Test*.class</include>
```

Comment:
Please note too that the default of "**/Test*.java **/*Test.java **/*TestCase.java" is not the same as "**/*Test*.class **/*Test.class". My include pattern includes MORE than the default, because it has *Test* in it.